awx/main/management/commands/create_preload_data.py

```diff
@@ -30,6 +30,7 @@ def handle(self, *args, **kwargs):
                 p = Project(name='Demo Project',
                             scm_type='git',
                             scm_url='https://github.com/ansible/ansible-tower-samples',
+                            scm_branch='master',
```

Comment:
I'd expect git to _default_ to master.  What error/problem are you seeing?

Reply:
@ryanpetrello for a fresh install, while trying to run a job template, it's not fetching the `hello_world.yml`. Initially I spent a lot of time figuring out what's wrong, and adding the branch ```master``` did the trick. This doesn't happen in creating a normal project, just once for the ```Demo Job Template```.